.travis.yml

```diff
@@ -2,23 +2,49 @@ language: php
 
 sudo: false
 
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+env:
+    global:
+        - TEST_COMMAND="composer test"
```

Comment:
what's the need for this global ?

Reply:
If we want to use test coverage. This will be changed. See example https://github.com/php-http/client-common/pull/90